### PR TITLE
Moving to robot-snapshots

### PR DESCRIPTION
**Pull request by pattacini**
This PR proposes the changes we agreed upon in order to be able to include in this repo also orphan branches corresponding to robots other than icub.

Once merged, I'll do the following:
- Update the name of the repo.
- Update the existing branch names to include the prefix `icub_`.

cc @icub-tech-iit/silo-mech

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
-$ git clone https://github.com/icub-tech-iit/icub-snapshots.git --single-branch --branch <branch-name>
+$ git clone https://github.com/icub-tech-iit/robot-snapshots.git --single-branch --branch <branch-name>
 ```
 
 ⚠ Be careful that branches are handled via [Git LFS](https://help.github.com/en/articles/installing-git-large-file-storage).
 
 
 ## Maintainers
-This repository is maintained by the CAD designer in the Team [PROTO](https://github.com/orgs/icub-tech-iit/teams/proto/members).
+This repository is maintained by the CAD designer in the Team PROTO.
```
Comment (pattacini) [inline README.md:21]: As this repo is public, links to internal resources shall be avoided.

File: README.md [hunk 2]
```diff
@@ -1,21 +1,21 @@
-iCub Snapshots
-==============
+Robot Snapshots
+===============
 
-This repository contains the snapshots for each robot version, S/N and/or batch of robots.
+This repository contains the CAD snapshots for each robot version, S/N and/or batch of robots.
```
Comment (pattacini) [inline README.md:4]:
```suggestion
This repository contains the snapshots for each robot version, S/N and/or batch of robots.
```

Actually, it's more generic. Let me restore the original version.

File: README.md [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
-$ git clone https://github.com/icub-tech-iit/icub-snapshots.git --single-branch --branch <branch-name>
+$ git clone https://github.com/icub-tech-iit/robot-snapshots.git --single-branch --branch <branch-name>
 ```
 
 ⚠ Be careful that branches are handled via [Git LFS](https://help.github.com/en/articles/installing-git-large-file-storage).
 
 
 ## Maintainers
-This repository is maintained by the CAD designer in the Team [PROTO](https://github.com/orgs/icub-tech-iit/teams/proto/members).
+This repository is maintained by the CAD designer in the Team PROTO.
```
Comment (pattacini) [inline README.md:21]:
```suggestion
The CAD designer of Team PROTO maintains this repository.
```

File: create_snapshot_branch.bat [hunk 4]
```diff
@@ -1,5 +1,6 @@
-set /p serialNumber="Enter the iCub S/N as three digits (e.g. 005): "
-git checkout --orphan sn_%serialNumber%
+set /p robotName="Enter the robot name in lower case (e.g. icub, ergocub): "
+set /p serialNumber="Enter the robot S/N as three digits (e.g. 005): "
+git checkout --orphan=%robotName%_sn_%serialNumber%
```
Comment (pattacini) [inline create_snapshot_branch.bat:3]:
```suggestion
git checkout --orphan %robotName%_sn_%serialNumber%
```

**Review by fiorisi — APPROVED** [review]
Thanks @pattacini, I've added a minor edit to the README.

**Comment by pattacini** [comment]
Minor comment. We're sticking to `robot` but in theory we could host here whatever mechatronic device we produce. I vote to stick to `robot` rather than `device` any how.

**Pull request merged**